src/ResourceManagement/Batch/Microsoft.Azure.Management.Batch/Generated/Models/BatchAccountUpdateParameters.cs

```diff
@@ -38,16 +38,10 @@ public partial class BatchAccountUpdateParameters
         /// account.</param>
         /// <param name="autoStorage">The properties related to auto storage
         /// account.</param>
-        /// <param name="poolAllocationMode">The allocation mode to use for
-        /// creating pools in the Batch account.</param>
-        /// <param name="keyVaultReference">A reference to the Azure key vault
-        /// associated with the Batch account.</param>
-        public BatchAccountUpdateParameters(IDictionary<string, string> tags = default(IDictionary<string, string>), AutoStorageBaseProperties autoStorage = default(AutoStorageBaseProperties), PoolAllocationMode? poolAllocationMode = default(PoolAllocationMode?), KeyVaultReference keyVaultReference = default(KeyVaultReference))
+        public BatchAccountUpdateParameters(IDictionary<string, string> tags = default(IDictionary<string, string>), AutoStorageBaseProperties autoStorage = default(AutoStorageBaseProperties))
```

Comment:
Need to update generate.cmd to refer to the spec using the commit for the recently merged PR.  Othewrwise this looks good